Integrations/integration-ArcherRSA.yml

```diff
@@ -2944,10 +2947,15 @@ script:
         var incidents = createIncidentsArr(results, dataArgs.applicationId);
 
         if (incidents.length > 0){
+            shouldAddOffset = true;
             lastTime = incidents[incidents.length-1].occurred;
-        }
 
-        setLastRun({ lastTime: lastTime + offset});
+            if (params.useEuropeanTime) {
+                lastTime = lastTime.getTime()
+            }
+        }
+        var newLastTime = shouldAddOffset ? lastTime + offset : lastTime;
```

Comment:
not sure i understood in which cases you will want to add the offset and why

Reply:
When you send the time from which you want to start fetching to Archer, you need to do adjustments so Archer would know the right time to start from (as there are differences between the time of your Archer instance and your local time).
However, we should only add the offset in two cases:
1. If Archer returns incidents, the time it returns is the time the incidents were created in ARCHER's time. Which means - you need to add the offset to get to the actual time they were created from your perspective.
2. If this is the first run you should add the offset, since it hasn't been added before

However, we shouldn't add the offset if there were no incidents fetched but this isn't our first run - since the offset is already included by induction.